test/suites/network_zone.sh

```diff
@@ -73,14 +69,18 @@ test_network_zone() {
   done
 
   # Setup DNS peers
-  lxc network zone set lxd.example.net peers.test.address=127.0.0.1
-  lxc network zone set lxdfoo.example.net peers.test.address=127.0.0.1 --project=foo
-  lxc network zone set 2.0.192.in-addr.arpa peers.test.address=127.0.0.1
-  lxc network zone set 0.1.0.1.2.4.2.4.2.4.2.4.2.4.d.f.ip6.arpa peers.test.address=127.0.0.1
+  lxc network zone set lxd.example.net peers.test.address=192.0.2.1
+  lxc network zone set lxdfoo.example.net peers.test.address=192.0.2.1 --project=foo
+  lxc network zone set 2.0.192.in-addr.arpa peers.test.address=192.0.2.1
+  lxc network zone set 0.1.0.1.2.4.2.4.2.4.2.4.2.4.d.f.ip6.arpa peers.test.address=192.0.2.1
```

Comment:
This `peers.test.address` is AFAIK the receiving end of a zone transfer (AXFR/IXFR) so `127.0.0.1` was a presumably a bogus placeholder that would simply drop those XFRs on the floor. In a real deployment, I'd expect a bind9/nsd server to get them.

Could we leave them at `127.0.0.1`? Or maybe I'm just reading it wrong.

Reply:
The transfer does not happen when I use 127.0.0.1

Reply:
This makes sense as the source addressed used by `dig` command will be the bridge's own IP and not 127.0.0.1

Reply:
@tomponline is this because there is an IP ACL that's derived from that `peers.test.address`? If do, would using `dig -b <address>` be a valid way to comply with the ACL?

I'm not trying to advocate to change anything in the test, just trying to wrap my mind around it (to maybe add some explaining notes in the test) :)

Reply:
Yeah, peers.test.address sets which peers can request the AXFR transfer.